src/mcp/grok_server.py

```
@@ -312,7 +356,16 @@ def _handle_grok_session_continue(params):
         log.error("Session %s error: %s", session_id, exc)
         return _error_content(f"Grok API error: {exc}")
 
-    return _text_content(strip_pgp_blocks(response))
+    cleaned = strip_pgp_blocks(response)
+    if write_files:
+        try:
+            written = parse_and_write_files(cleaned, output_dir)
+        except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)